src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,47 @@
+package duke.parser;
+
+import static duke.taskList.taskListOperations.*;
+import static duke.common.Messages.*;
+import static duke.ui.UserInterface.*;
+
+public class Parser {
+    /**
+     * Special condition for each commands done, todo, deadline, event.
```

Comment:
The introductory line can first tell what this class/method is about.
E.g., `Parses the given input to identify the command`. This can be followed by the description you have.